README.md

```diff
@@ -98,6 +98,49 @@ jobs:
 4. Add `github_deployment` to the docker group: `sudo usermod -aG docker github_deployment`
 5. Create the deployment directory `/opt/github` and give `github_deployment` access: `sudo mkdir /opt/github && sudo chown github_deployment:github_deployment /opt/github`
 6. Switch to `github_deployment` user: `sudo su github_deployment`
+> <details>
+> <summary>Note on fixing “access denied” when switching to deployment user</summary>
+>
+> If, when you run:
+> ```bash
+> sudo su github_deployment
+> ```
+> you see an error like:
+> ```bash
+> su: cannot open session: Permission denied
+> pam_access(su:session): access denied for user `github_deployment' from `pts/…'
+> ```
+> then PAM is blocking `github_deployment` because of an `/etc/security/access.conf` rule. Some VMs ship with this file empty (no restrictions), while others have existing allow/deny rules—either way, you need to ensure there is a `+` line permitting our deployment user before any `deny all`.
+> - Open `/etc/security/access.conf` (as root):
+> ```bash
+> sudo vi /etc/security/access.conf
+> ```
+> - Add the following line:
+> ```bash
+> # Allow github_deployment user to access the system
+> +:github_deployment : ALL
+> ```
```

Comment:
Please add a note that this introduces some security risks. It is better to restrict the deployment user to only the required commands

Reply:
Just place that as a comment - then i'm ok with it